modin/data_management/factories/factories.py

```diff
@@ -60,152 +107,147 @@ def get_info(cls) -> FactoryInfo:
         )
 
     @classmethod
-    def prepare(cls):
-        """
-        Fills in .io_cls class attribute lazily
-        """
+    @doc(_doc_factory_prepare_method, engine_name="an underlying")
+    def prepare(cls):  # noqa: D102
```

Comment:
`D102: missing docstring`, we're inheriting docs here, doc-checker script can't handle that case